mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/bridge/mapping/annotation/RoutingBinderRef.java

```diff
@@ -41,6 +43,12 @@
 	 */
 	BeanRetrieval retrieval() default BeanRetrieval.ANY;
 
+	/**
+	 * @return A parameter that can be statically passed to a given {@link IdentifierBinder},
+	 * so that it can be used by its {@link IdentifierBridge}.
+	 */
+	Parameter[] params() default {};
```

Comment:
I noticed the use of `params` here (abbreviated) but `parameter` to retrieve the values. Let's decide on using abbrieviations or not, and stick to it? I vote for abbreviations everywhere, since that's what we do for named predicates already.

Reply:
I vote for the same. Let's rename `Parameter` to `Param`.